### PR TITLE
Feat: Added wishlist page

### PR DESCRIPTION
**Pull request by sharmivijay**
kindly review and add your comment.

### CODE REVIEW

File: styles/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+.card-text i{
+    padding: 0.5rem;
+}
+
+.disabled
+{
+    cursor:no-drop;
+    opacity: 0.5; 
+}
+.red-txt{
```
Comment (shmbajaj53) [inline styles/wishlist.css]: Use beautify to format code.

File: styles/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
+body{
+    margin: 0px;
+}
+.box1{ grid-area: header; }
```
Comment (bajajcodes) [inline styles/wishlist.css]: Class names can be meaningful and self-descriptive.

File: styles/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    'box2 box2 box2 box2 box2 box2'
+    'footer footer footer footer footer footer';
+  gap: 1rem;
+  margin-bottom: 0%;
+  background-color: var(--success-green);
+}
+
+
+.responsive-img{
+    width: 100%;
```
Comment (bajajcodes) [inline styles/wishlist.css]: You can use either of width or max-width depending upon your use case, using both is un-necessary.

File: styles/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    margin-bottom: 0%;
+    background-color: yellowgreen;
+}
+
+
+.grid-container {
+  display: grid;
+  grid-template-areas:
+    'header header header header header header'
+    'box2 box2 box2 box2 box2 box2'
```
Comment (bajajcodes) [inline styles/wishlist.css]: Meanigful class names helps to improve code readbility.

**Review by bajajcodes — COMMENTED** [review]
- Use CL utilities from your component library, if not having then define static or commonly used values in root of css file such as 1rem.
- Give meanigful class names to improve code readbility.
- Use some extension to make code look clean.

File: styles/wishlist.css (hunk 3 above)
Comment (sharmivijay) [reply styles/wishlist.css]: sure. i will update it

File: styles/wishlist.css (hunk 2 above)
Comment (sharmivijay) [reply styles/wishlist.css]: sure. i will update the class names

**Pull request merged**